utils/getInvoices/tests/getInvoices.garbage-collector.test.ts

```diff
@@ -5,142 +5,661 @@ import { expect } from '@jest/globals'
+      it('should NOT load when service = null, company = null', () => {
+        const service: Partial<IService> = null
+        const company: Partial<IRealestate> = null
+
+        const invoices = getInvoices({
+          service,
+          company,
+        })
+
+        expect(invoices).not.toContainEqual(
+          expect.objectContaining({ type: ServiceType.GarbageCollector })
+        )
+      })
+      it('should NOT load when service = { garbageCollectorPrice: 10 }, company = { rentPart: 10, garbageCollector: false }', () => {
+        const service: Partial<IService> = {
+          garbageCollectorPrice: 10,
+        }
+        const company: Partial<IRealestate> = {
+          rentPart: 10,
+          garbageCollector: false,
+        }
+
+        const invoices = getInvoices({
+          service,
+          company,
+        })
+
+        expect(invoices).not.toContainEqual(
+          expect.objectContaining({ type: ServiceType.GarbageCollector })
+        )
+      })
+      it('should load when service = { garbageCollectorPrice: 10 }, company = { rentPart: 10, garbageCollector: true }', () => {
+        const service: Partial<IService> = {
+          garbageCollectorPrice: 10,
+        }
+        const company: Partial<IRealestate> = {
+          rentPart: 10,
+          garbageCollector: true,
+        }
+
+        const invoices = getInvoices({
+          service,
+          company,
+        })
+
+        expect(invoices).toContainEqual({ 
+          type: ServiceType.GarbageCollector,
+          price: 1,
+          sum: 1, 
+        })
+      })
+      it('should load when service = { garbageCollectorPrice: 0 }, company = { rentPart: 0, garbageCollector: true }', () => {
+        const service: Partial<IService> = {
+          garbageCollectorPrice: 0,
+        }
+        const company: Partial<IRealestate> = {
+          rentPart: 0,
+          garbageCollector: true,
+        }
+
+        const invoices = getInvoices({
+          service,
+          company,
+        })
+
+        expect(invoices).toContainEqual({ 
           type: ServiceType.GarbageCollector,
           price: 0,
-          sum: 110,
-        },
-      ],
-    }
-    const prevPayment: Partial<IPayment> = null
-
-    const invoices = getInvoices({
-      company,
-      service,
-      payment,
-      prevPayment,
-    })
+          sum: 0,
+        })
+      })
+      it('should NOT load when service = { garbageCollectorPrice: 10 }, company = null', () => {
+        const service: Partial<IService> = {
+          garbageCollectorPrice: 10,
+        }
+        const company: Partial<IRealestate> = null
+
+        const invoices = getInvoices({
+          service,
+          company,
+        })
+
+        expect(invoices).not.toContainEqual(
+          expect.objectContaining({ type: ServiceType.GarbageCollector })
+        )
+      })
+      it('should NOT load when service = { garbageCollectorPrice: 10 }, company = { discount: 0 }', () => {
```

Comment:
rename to `should NOT load when service = { garbageCollectorPrice: 10 }, company = { ... }`